app/controllers/insured/families_controller.rb

```diff
@@ -293,20 +293,31 @@ def init_qualifying_life_events
     else
       if @person.active_employee_roles.present?
         if current_user.has_hbx_staff_role?
-          @qualifying_life_events += QualifyingLifeEventKind.shop_market_events_admin
+          @qualifying_life_events += fetch_Only_SHOP_QLE
         else
           @qualifying_life_events += QualifyingLifeEventKind.shop_market_events
         end
       else @person.consumer_role.present?
         if current_user.has_hbx_staff_role?
-          @qualifying_life_events += QualifyingLifeEventKind.individual_market_events_admin
+          @qualifying_life_events += fetch_Only_SHOP_QLE
         else
           @qualifying_life_events += QualifyingLifeEventKind.individual_market_events
         end
       end
     end
   end
 
+  # This method returns shop market QLE when dependent house hold adding or removing
+  #
+  # @return [Array<BSON_FIELDS>]
+  def fetch_Only_SHOP_QLE
```

Comment:
My one thought is that maybe this method belongs in the QualifyingLifeEventKind model.

Maybe something like QualifyingLifeEventKind.fetch_applicable_market_events_admin --- and have the conditional in that method.

Just a thought - we can discuss tomorrow